scripts/test/test_install.ps1

```diff
@@ -40,15 +41,14 @@ $exclude_tests = @("flarevm.installer.vm")
 $built_pkgs = Get-ChildItem $built_pkgs_dir | Foreach-Object { ([regex]::match($_.BaseName, '(.*?[.](?:vm)).*').Groups[1].Value) -and $_ -notin $exclude_tests}
 Set-Location $built_pkgs_dir
 foreach ($package in $built_pkgs) {
-    choco install $package -y -s "'.;https://www.myget.org/F/vm-packages/api/v2;https://community.chocolatey.org/api/v2/'" --no-progress --force
-    if ($validExitCodes -notcontains $LASTEXITCODE) { # Abort with the first failing install
-        Write-Host -ForegroundColor Red "[ERROR] Failed to install $package"
-        # Restore the original location
-        Set-Location -Path $org_dir -PassThru | Out-Null
-        Exit 1
+    choco install $package -y -r -s "'.;https://www.myget.org/F/vm-packages/api/v2;https://community.chocolatey.org/api/v2/'" --no-progress --force
+    if ($validExitCodes -notcontains $LASTEXITCODE) {
+        $failed = $true
+        if (-not $all) { break } # Abort with the first failing install
```

Comment:
```suggestion
        if (-not $all.IsPresent) { break } # Abort with the first failing install
```

Reply:
It works without the `IsPresent`. I see on the internet that people use `IsPresent` `-eq $true` and nothing, is there a reason we prefer one of this options?

Reply:
To me it reads better to have `<switch_variable_name>.IsPresent`